<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company Name</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #3a3a3a;
	color: #fff;
	selection-background-color: #b78620;
	selection-color: #000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QMenuBar-----*/
QMenuBar 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
	color: #fff;

}


QMenuBar::item 
{
	background-color: transparent;

}


QMenuBar::item:selected 
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;

}


QMenuBar::item:pressed 
{
	background-color: rgb(183, 134, 32);
	border: 1px solid #b78620;
	color: #fff;

}


/*-----QMenu-----*/
QMenu
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    border: 1px solid #222;
    padding: 4px;
	color: #fff;

}


QMenu::item
{
    background-color: transparent;
    padding: 2px 20px 2px 20px;

}


QMenu::separator
{
   	background-color: rgb(183, 134, 32);
	height: 1px;

}


QMenu::item:disabled
{
    color: #555;
    background-color: transparent;
    padding: 2px 20px 2px 20px;

}


QMenu::item:selected
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;

}


/*-----QToolBar-----*/
QToolBar
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(69, 69, 69, 255),stop:1 rgba(58, 58, 58, 255));
	border-top: none;
	border-bottom: 1px solid #4f4f4f;
	border-left: 1px solid #4f4f4f;
	border-right: 1px solid #4f4f4f;

}


QToolBar::separator
{
	background-color: #2e2e2e;
	width: 1px;

}


/*-----QToolButton-----*/
QToolButton 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;
}


QToolButton:hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;
	
}


QToolButton:pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #b78620;

}


QToolButton:checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #222;
}


/*-----QPushButton-----*/
QPushButton
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::flat
{
	background-color: transparent;
	border: none;
	color: #fff;

}


QPushButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QPushButton::hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;

}


QPushButton::pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #b78620;

}


QPushButton::checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(74, 74, 74, 255),stop:1 rgba(49, 49, 49, 255));
	border: 1px solid #222;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QPlainTExtEdit-----*/
QPlainTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}

/*-----QTExtEdit-----*/
QTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QTabBar-----*/
QTabBar::tab
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-color: #666;
	border-bottom: none;
	padding: 5px;
	padding-left: 15px;
	padding-right: 15px;

}


QTabWidget::pane 
{
	background-color: red;
	border: 1px solid #666;
	top: 1px;

}


QTabBar::tab:last
{
	margin-right: 0; 

}


QTabBar::tab:first:!selected
{
	background-color: #0c0c0d;
	margin-left: 0px;

}


QTabBar::tab:!selected
{
	color: #b1b1b1;
	border-bottom-style: solid;
	background-color: #0c0c0d;

}


QTabBar::tab:selected
{
	margin-bottom: 0px;

}


QTabBar::tab:!selected:hover
{
	border-top-color: #b78620;

}


/*-----QComboBox-----*/
QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
    border: 1px solid #000;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #b78620;
	color: #000;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b78620;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;

}


QComboBox::down-arrow
{
    image: url(icons/arrow-down.png);
    width: 14px;
    height: 14px;
	padding-top: 4px;
}


/*-----QSpinBox &amp; QDateTimeEdit-----*/
QSpinBox,
QDateTimeEdit 
{
    background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	padding: 3px;
	padding-left: 5px;
    border-radius : 2px;

}


QSpinBox::up-button, 
QDateTimeEdit::up-button
{
	border-top-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-button:hover, 
QDateTimeEdit::up-button:hover
{
	background-color: #585858;

}


QSpinBox::up-button:pressed, 
QDateTimeEdit::up-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;
    height: 7px;

}


QSpinBox::down-button, 
QDateTimeEdit::down-button
{
	border-bottom-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-button:hover, 
QDateTimeEdit::down-button:hover
{
	background-color: #585858;

}


QSpinBox::down-button:pressed, 
QDateTimeEdit::down-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(icons/arrow-down.png);
    width: 7px;
    height: 7px;

}


/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #666666;
	border-radius: 5px;
    margin-top: 20px;

}


QGroupBox::title  
{
    background-color: transparent;
    color: #eee;
    subcontrol-origin: margin;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*-----QHeaderView-----*/
QHeaderView::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
    color: #fff;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


QTableCornerButton::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;

}


/*-----QTreeWidget-----*/
QTreeView
{
	show-decoration-selected: 1;
	alternate-background-color: #3a3a3a;
	selection-color: #fff;
	background-color: #2d2d2d;
	border: 1px solid gray;
	padding-top : 5px;
	color: #fff;
	font: 8pt;

}


QTreeView::item:selected
{
	color:#fff;
	background-color: #b78620;
	border-radius: 0px;

}


QTreeView::item:!selected:hover
{
    background-color: #262626;
    border: none;
    color: white;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}


QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QListView-----*/
QListView 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(83, 83, 83, 255),stop:0.293269 rgba(81, 81, 81, 255),stop:0.634615 rgba(79, 79, 79, 255),stop:1 rgba(83, 83, 83, 255));
    border : none;
    color: white;
    show-decoration-selected: 1; 
    outline: 0;
	border: 1px solid gray;

}


QListView::disabled 
{
	background-color: #656565;
	color: #1b1b1b;
    border: 1px solid #656565;

}


QListView::item 
{
	background-color: #2d2d2d;
    padding: 1px;

}


QListView::item:alternate 
{
    background-color: #3a3a3a;

}


QListView::item:selected 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:selected:!active 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:selected:active 
{
	background-color: #b78620;
	border: 1px solid #b78620;
	color: #fff;

}


QListView::item:hover {
    background-color: #262626;
    border: none;
    color: white;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
    color: lightgray;
	border: none;

}


QCheckBox::indicator
{
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: #b78620;
    border: 1px solid #3a546e;

}


QCheckBox::indicator:unchecked:hover
{
	border: 1px solid #b78620; 

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QRadioButton-----*/
QRadioButton 
{
	color: lightgray;
	background-color: transparent;

}


QRadioButton::indicator::unchecked:hover 
{
	background-color: lightgray;
	border: 2px solid #b78620;
	border-radius: 6px;
}


QRadioButton::indicator::checked 
{
	border: 2px solid #b78620;
	border-radius: 6px;
	background-color: rgba(183,134,32,20%);  
	width: 9px; 
	height: 9px; 

}


/*-----QSlider-----*/
QSlider::groove:horizontal 
{
	background-color: transparent;
	height: 3px;

}


QSlider::sub-page:horizontal 
{
	background-color: #b78620;

}


QSlider::add-page:horizontal 
{
	background-color: #131313;

}


QSlider::handle:horizontal 
{
	background-color: #b78620;
	width: 14px;
	margin-top: -6px;
	margin-bottom: -6px;
	border-radius: 6px;

}


QSlider::handle:horizontal:hover 
{
	background-color: #d89e25;
	border-radius: 6px;

}


QSlider::sub-page:horizontal:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:horizontal:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:horizontal:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    height: 15px;
    margin: 0px 16px 0 16px;

}


QScrollBar::handle:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;

}


QScrollBar::add-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: right;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: left;
    subcontrol-origin: margin;

}


QScrollBar::right-arrow:horizontal
{
    image: url(://arrow-right.png);
    width: 6px;
    height: 6px;

}


QScrollBar::left-arrow:horizontal
{
    image: url(://arrow-left.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;

}


QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 16px;
	border: 1px solid #2d2d2d;
    margin: 16px 0px 16px 0px;

}


QScrollBar::handle:vertical
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;

}


QScrollBar::add-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}


QScrollBar::up-arrow:vertical
{
    image: url(://arrow-up.png);
    width: 6px;
    height: 6px;

}


QScrollBar::down-arrow:vertical
{
    image: url(icons/arrow-down.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;

}


/*-----QProgressBar-----*/
QProgressBar
{
    border: 1px solid #666666;
    text-align: center;
	color: #000;
	font-weight: bold;

}


QProgressBar::chunk
{
    background-color: #b78620;
    width: 30px;
    margin: 0.5px;

}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name Surname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generate receipt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_notax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total (no tax)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adress, house number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Postal code, City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="postalcode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>395</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Receipt number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiptID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>415</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generic default item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Items description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description_receipt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;"Default generic item description"&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="currency">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string>EUR</string>
    </property>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HRK</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>505</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total (with tax)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amount_tax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>525</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="IDwarning">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: red;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Receipt number already exists!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>415</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(icons/calc.png);
height: 25px;
width: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Personal identification number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oib">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Settings"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>address</tabstop>
  <tabstop>postalcode</tabstop>
  <tabstop>oib</tabstop>
  <tabstop>description</tabstop>
  <tabstop>description_receipt</tabstop>
  <tabstop>receiptID</tabstop>
  <tabstop>amount_notax</tabstop>
  <tabstop>amount_tax</tabstop>
  <tabstop>calc_btn</tabstop>
  <tabstop>currency</tabstop>
  <tabstop>generate_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
